gateway/src/main/resources/application.yml

```diff
@@ -19,6 +19,9 @@ server:
 spring:
   config:
     import: optional:file:${georchestra.datadir}/default.properties,optional:file:${georchestra.datadir}/gateway/gateway.yaml
+  messages:
+    basename: messages/login
```

Comment:
for now on, we only have the login page to customize with translations. I think it should be ok for now, but might need to revisit later on (logout page ?)